packages/web/components/Modal/CreateInstitutionsModal/StepTwo.js

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { InputField, MaskedInputField, SelectField } from '../../Form';
+import { Cell, Row } from '../../Common';
+import { beforeMaskedValueChange } from '../../../utils/helper';
+
+const typeOptions = [
+	{
+		label: 'Público',
+		value: 'public',
+	},
+	{
+		label: 'Privado',
+		value: 'private',
+	},
+	{
+		label: 'Misto',
+		value: 'mixed',
+	},
+	{
+		label: 'Outro',
+		value: 'other',
+	},
+];
+
+const categoryOptions = [
+	{
+		label: 'Universidade',
+		value: 'university',
+	},
+	{
+		label: 'Instituto',
+		value: 'institute',
+	},
+	{
+		label: 'Associação',
+		value: 'association',
+	},
+	{
+		label: 'Fundação',
+		value: 'foundation',
+	},
+	{
+		label: 'Cooperativa',
+		value: 'cooperative',
+	},
+	{
+		label: 'Companhia',
+		value: 'company',
+	},
+	{
+		label: 'Outro',
+		value: 'other',
+	},
+];
+
+const StepTwo = ({ form }) => {
+	const { setValue } = form;
+
+	return (
+		<>
+			<Row>
+				<Cell col={12}>
+					<InputField
+						form={form}
+						label="E-mail para contato"
+						name="email"
+						type="email"
+						variant="gray"
+					/>
+				</Cell>
+			</Row>
+			<Row>
+				<Cell col={6}>
+					<MaskedInputField
+						form={form}
+						name="phone_number"
+						alwaysShowMask={false}
+						label="Telefone"
+						mask="(99) 9999-99999"
+						maskChar={null}
+						beforeMaskedValueChange={beforeMaskedValueChange}
+						pattern={/(\(?\d{2}\)?\s)?(\d{4,5}-\d{4})/}
```

Comment:
We must extract this pattern to a variable in `utils`. What do you think about that?

```suggestion
						pattern={maskPattern.phoneNumber}
```

Reply:
@mateus4k good idea!